### DISCUSSION TITLE
Copilot for Visual Studio 2022 says it's not officially supported?

### DISCUSSION THREAD

**Discussion by dmeatte-aq** [community/community#113444] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I asked what the shortcuts are in vs 2022, and the ai says the extension isn't GA and then lists out the default shortcuts for VS code. `/help` does the right thing.

![image](https://github.com/community/community/assets/44836981/a71f64c3-702b-4805-a7bb-b1a23ea2672f)

**Comment by ayim** [comment]
Hi @dmeatte-aq, thanks for reporting this.
I've created a tracking bug here: https://developercommunity.visualstudio.com/t/Copilot-for-Visual-Studio-2022-says-its/10620549
